cylc/flow/network/scan.py

```diff
@@ -316,6 +323,17 @@ async def contact_info(flow):
     return flow
 
 
+@pipe
+async def validate_contact_info(flow):
+    """Return flow if contact file is valid"""
```

Comment:
```suggestion
    """Return flow if contact file is valid.
    
    This checks that the contact file contains the minimal set of fields required
    for most purposes.
    
    This helps to protect against blank or corrupted (yet valid) contact files
    which would otherwise pass through without raising errors.
    """
```